multimedia/tvheadend/Makefile

```diff
@@ -1,22 +1,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tvheadend
-PKG_VERSION:=2022-11-20
-PKG_RELEASE:=3
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=LICENSE.md
 
+PKG_RELEASE:=4
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/tvheadend/tvheadend.git
-PKG_MIRROR_HASH:=9c1bb3eea3f3539454d17e4c6aabc774a104a772aa34ed39f248521cf6488ce5
-PKG_SOURCE_VERSION:=0ff96106aa2e0f9a384c3a2662ca005797a6b399
-PKG_SOURCE_DATE:=2022-11-20
-
-PKG_LICENSE:=GPL-3.0
-PKG_LICENSE_FILES:=LICENSE.md
+PKG_SOURCE_VERSION:=003fd92707531bdf7ad1753ab028db8748ac5ab8
+PKG_MIRROR_HASH:=ff4b21ebb2250a327215489095561ed2ed6a6505cdee0835c2b27b54e5418944
+PKG_SOURCE_DATE:=2023-06-05
```

Comment:
Why you changed order? The previous one was alright

Reply:
It's mostly unintentionally. I made a lot of changes trying to fix this thing:
https://forum.openwrt.org/t/pkg-mirror-hash-is-always-wrong/163182
At some point I even tried using the github zip download instead of git clone. I didn't take the extra care to put every line in the exact same order again. Does it matter?

Reply:
I can not say that it does not matter. We are trying to have some row order here as it makes it easier to review things, etc. If we say that it does not matter then it would look really bad as one thing is up, another thing is down. It just complicate things. Yes, someone can say that it is purely cosmetic, it is of course, but don't introduce such changes if they are not necessary and in your case, it is not.

Reply:
Ok. Resolved.